.github/workflows/ci.yaml

```diff
@@ -96,8 +104,28 @@ jobs:
 
       - name: Run the playbook
         run: |
+          read -r -d '' extra_vars << 'EOF' || true
```

Comment:
Dark magic. :magic_wand: Apparently this is a shell builtin, and in `fish` it's different.

Reply:
Yeah... I was just struggling to find a different way how to pass that many variables to ansible w/o creating a temporary YAML file :( We could do that instead to make this portable across different shells.